### ISSUE TITLE
Watching Server Crash (MC 1.10.2)

### ISSUE THREAD

**Issue by DMIL421** [raoulvdberge/refinedstorage#1103]
#### Issue description:
I woke up this morning to a line of crashlogs in the server's folder. Description of the Crash is a Watching Server, and appears to be caused by something with Refined Storage. At first, I thought this might be a power-flicker issue as we have had those before, but temporarily disabling RS's need for power has not resolved it. The server continues to crash on startup. All logs are identical. I honestly don't know what I'm looking for, so I'm reaching out.
Please let me know if there is any other info I can offer or any way I can help.

#### What happens:
Server starts, loads, and instantly crashes.

#### What you expected to happen:
The server not crash? :P
#### Steps to reproduce:
1. I'm afraid I don't understand what is happening enough to say how to reproduce. Hopefully the crashlog helps.
I did try to revert RS versions to .24 but received the same crash.

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 12.18.3.2239
- Refined Storage: 1.2.25

Does this issue occur on a server? [yes/no]
Yes
#### If a (crash)log is relevant for this issue, link it here:
https://gist.github.com/DMIL421/59a8ddb044fb40279395ce2b95a9ad63

**Comment by raoulvdberge** [comment]
This is very weird. At first glance, this does look like a power issue (power going on - off constantly and rebuilding the network - causing lag) - but - watchdog crashes can be unreliable so I'm not sure what I'm looking at here.

If you disable RS entirely, does it still crash?

**Comment by DMIL421** [comment]
Disable, as in remove entirely? Or are you talking about a config change? I can have the guy who runs the server pull a backup. Would you like a copy of the world and pack for test? I can link both.

**Comment by raoulvdberge** [comment]
Remove RS from the mods folder (or your pack) entirely (both server and client) and see if it still causes trouble.

Take a backup before you do this, so you can revert, obviously.

If you still have problems, this is not an RS issue. If you don't have problems anymore, this is in fact an RS issue and we'll have to look into this further.

**Comment by DMIL421** [comment]
Will do.

**Comment by DMIL421** [comment]
Okay, pulled RS, Growable Cells, and Wireless Crafting Terminal. Server started up without issue. Since I am guessing you will ask, I'll have the save reverted and pull only Wireless Crafting Terminal and Growable Cells to make sure they were not the problem. I doubt it, but it is worth checking.

**Comment by DMIL421** [comment]
Okay... so that's interesting... this time the server also started with RS but without those two files. Gonna do some checking to see if I can figure out which was crashing. I'll update when I know for sure, but appears to not be a RS issue.

**Comment by DMIL421** [comment]
Oh, looks like it started using the .24 version that I had used as a test to get it working. Updated to .25 and it crashed.

**Comment by raoulvdberge** [comment]
I think the version here is irrelevant, the crashes appear random.

**Comment by DMIL421** [comment]
Having trouble getting the current crashlog... I'll update when I do. I'll also confirm that it starts properly with .25 since the successful tests were .24.

**Comment by DMIL421** [comment]
I think I found the problem. I removed the chunkloading mod that I had in the pack from the server and I was able to get it started. I was also able to load in.
I realized that I had an RS secondary network that was receiving power from a Draconic Wireless Energy Crystal that could have been delayed in providing power to the controller resulting in a flickering issue. I've removed that sub-network and will be watching for any further crashes. If I re-add that network, I will likely set up a Draconic i/o crystal as it will have an internal capacity that can ensure that this problem doesn't happen again. My main network is powered in this way and has not been a problem.
Thanks for looking at this @raoulvdberge. I appreciate all your work.
